dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/timers/Timer.xtxtuml

```diff
@@ -28,18 +35,38 @@
 	 *            the signal which is to be sent after the delay
 	 * @param millisecs
 	 *            the time in millisecs to wait before sending the signal
-	 * @return a timer instance to manage this delayed send operation before it
+	 * @return a handle object to manage this delayed send operation before it
 	 *         happens
 	 */
-	static Timer start(ModelClass targetObj, Signal signal, int millisecs) {
-		return new TimerImpl(targetObj, signal, millisecs);
+	public static external-body Timer start(Signal sig, ModelClass targetObj, int millisecs) {
```

Comment:
> The `create` here is the `Action.create` API method which in the exported UML2 model is an object creation action followed by a constructor call. However, the constructor which is called here is external.

So if I understand it correctly, the `external-body` modifier is required on `start` because `Action.create` is actually unfolded in place into two actions by the model exporter, where one of these actions is external, as it calls an external constructor.

Does this imply that even though `Action.create` is labeled as `external-body` (i.e. it should be safe to call it from pure action code), the user will have to manually check the implicitly called constructor and mark the the enclosing operation with `external-body` if it's external? If yes, isn't this counter-intuitive?